boards/x86/qemu_x86/Kconfig.board

```diff
@@ -6,3 +6,4 @@ config BOARD_QEMU_X86
 	select HAS_DTS
 	select HAS_DTS_ETHERNET
 	select CPU_HAS_FPU if !X86_IAMCU
+	select HAS_COVERAGE_SUPPORT
```

Comment:
qemu_cortex_m3 is notably not in the list of boards that selects HAS_COVERAGE_SUPPORT.

Reply:
qemu_cortex_m3 will be phased out soon. 
mps2_an385 will be used in place of qemu_cortex_m3.